dist/pathfora.js

```diff
@@ -626,7 +628,7 @@
 
       if (valid && core.valid) {
         sessionStorage.setItem(id, sessionImpressions);
-        utils.saveCookie(id, Math.min(totalImpressions, 9998) + "," + now);
+        utils.saveCookie(id, Math.min(totalImpressions, 9998) + "|" + now);
```

Comment:
this isn't hurting n e thing, leaving for now but when we do a cleanup im pretty sure it can go.